udata/translations/fr/LC_MESSAGES/udata.po

```diff
@@ -1394,97 +1318,65 @@ msgstr "Cela doit retourner une réponse de ce style :"
 
 #: udata/templates/apidoc.html:133
 msgid ""
-"This is very verbose and we don't need all that information yet. That's "
-"why we use jq.\n"
+"This is very verbose and we don't need all that information yet. That's why we use jq.\n"
 "                "
-msgstr ""
-"C'est très verbeux et nous n'avons pas besoin de toute cette information "
-"pour l'instant. C'est la raison pour laquelle nous utilisons jq."
+msgstr "C'est très verbeux et nous n'avons pas besoin de toute cette information pour l'instant. C'est la raison pour laquelle nous utilisons jq."
 
 #: udata/templates/apidoc.html:136
 msgid "Verifying that jq is working"
 msgstr "Vérifier que jq fonctionne"
 
 #: udata/templates/apidoc.html:138
 msgid ""
-"Once jq is installed, you can verify that it works as expected by typing "
-"that command in your shell:\n"
+"Once jq is installed, you can verify that it works as expected by typing that command in your shell:\n"
 "                "
-msgstr ""
-"Une fois jq installé, vous pouvez vérifier qu'il fonctionne en tapant "
-"cette commande dans votre terminal :"
+msgstr "Une fois jq installé, vous pouvez vérifier qu'il fonctionne en tapant cette commande dans votre terminal :"
```

Comment:
No, it's only translations, not html. If we want to had some markup, it should be in the template.
This page is particular. I had some work miving it to markdown allowing to have one file by language.